spec/spec_helper.rb

```diff
@@ -48,6 +48,21 @@ def coverage_needed?
 Capybara.default_driver = :rack_test      # This is a faster driver
 Capybara.javascript_driver = :poltergeist # This is slower
 Capybara.default_max_wait_time = ENV['TRAVIS'] ? 30 : 15
+# Adding the below to deal with random Capybara-related timeouts in CI.
+# Found in this thread: https://github.com/teampoltergeist/poltergeist/issues/375
+poltergeist_options = {
+  js_errors: false,
```

Comment:
While this will squelch the false-positives, some of the js errors are legitimate. This has bitten us several times and we've now set it to `true` because there were errors we weren't seeing until we had deployed the app to our QA environment.